app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -7,50 +7,151 @@ class RefereeProfile extends Component {
     match: PropTypes.shape({
       params: PropTypes.object
     }).isRequired
-  }
+  };
 
   state = {
+    httpStatus: 200,
+    httpStatusText: '',
     firstName: '',
     lastName: '',
     bio: '',
-    email: ''
-  }
+    email: '',
+    pronouns: '',
+    nationalGoverningBodies: [],
+    certifications: []
+  };
 
   componentDidMount() {
-    const { match: { params } } = this.props
+    const { match: { params } } = this.props;
 
     axios.get(`/api/v1/referees/${params.id}`)
-      .then(({ data }) => {
-        const { attributes } = data.data
+      .then(({ status, statusText, data }) => {
```

Comment:
Let's wrap this logic in a helper function, so something like:

```
setDataFromFetch = (status, statusText, data) => {
  const { data: { attributes }, included } = data
  ...
}
```

so this would become:

```
.then(({ status, statusText, data }) => this.setFetchData(status, statusText, data))
```

This way we can change the internals of that function without the componentDidMount having to be concerned with what is going on.